### PR TITLE
add tab count badge

### PR DESCRIPTION
**Pull request by insaner**
Please let me know if I did this right, I'm just starting to figure out how github/git work.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -20,5 +20,10 @@
 	],
 	"web_accessible_resources": [
 		"images/*"
-	]
+	],
+
```
Comment (RJHsiao) [inline manifest.json]: Please remove empty line

File: js/eventPage.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// added from "chrome tab count"
+
+function updateIcon(tabId, changeInfo, tab) {
+    chrome.tabs.query({currentWindow: true},
+    // chrome.tabs.query({},
```
Comment (RJHsiao) [inline js/eventPage.js]: Please remove unused code (Lin5), empty lines (L7 & L9), and align indent (L6~12).

**Review by RJHsiao — CHANGES_REQUESTED** [review]
Please try to resolve about following comments.

**Comment by insaner** [comment]
Ok, I hope I did it right. Everything should look ok now.

**Review by RJHsiao — APPROVED** [review]

**Comment by RJHsiao** [comment]
Well done!

**Comment by insaner** [comment]
Awesome! Thank you!

**Pull request merged**